src/components/BarChart/examples/04.grouped-with-legend.jsx

```diff
@@ -2,16 +2,19 @@
 
 import _ from 'lodash';
 import React from 'react';
-import {
-	BarChart,
-	Legend,
-	chartConstants,
-} from '../../../index';
+import { BarChart, Legend, chartConstants } from '../../../index';
 
 const data = [
-	{ x: 'Monday'    , apples: 10 , pears: 20 , peaches: 35 , bananas: 15 , oranges: 5  } ,
-	{ x: 'Tuesday'   , apples: 20 , pears: 5  , peaches: 20 , bananas: 25 , oranges: 27 } ,
-	{ x: 'Wednesday' , apples: 5  , pears: 15 , peaches: 5  , bananas: 20 , oranges: 35 } ,
+	{ x: 'Monday', apples: 10, pears: 20, peaches: 35, bananas: 15, oranges: 5 },
+	{ x: 'Tuesday', apples: 20, pears: 5, peaches: 20, bananas: 25, oranges: 27 },
+	{
```

Comment:
i kinda like the 80 char width, i'm ok with this since it's kind of an edge case. would be kinda nice if we could opt out of it for this one line, but meh